pkg/controllers/dynakube/extension/eec/statefulset.go

```diff
@@ -353,6 +379,27 @@ func setVolumes(dk *dynakube.DynaKube) func(o *appsv1.StatefulSet) {
 			})
 		}
 
+		if dk.Spec.Templates.ExtensionExecutionController.TlsRefName != "" {
+			o.Spec.Template.Spec.Volumes = append(o.Spec.Template.Spec.Volumes, corev1.Volume{
+				Name: eecTlsCertificatesVolumeName,
+				VolumeSource: corev1.VolumeSource{
+					Secret: &corev1.SecretVolumeSource{
+						SecretName: dk.Spec.Templates.ExtensionExecutionController.TlsRefName,
```

Comment:
just use this name in a condition in line 334

This Volume duplicates 330-348

Reply:
on it.